vizro-core/tests/unit/vizro/models/test_utils.py

```diff
@@ -0,0 +1,248 @@
+import textwrap
+from typing import List, Optional
+
+import pytest
+import vizro.models as vm
+import vizro.plotly.express as px
+from vizro.actions import export_data
+from vizro.models._utils import _concatenate_code
+from vizro.models.types import capture
+
+extra_imports_string = "import vizro.models as vm\nimport pandas as pd"
+code_string = "vm.Card(text='Foo')"
+callable_defs_string = """def f():\n  return 'hi'"""
+data_settings_string = """# data_manager["iris"] = ===> Fill in here <==="""
+
+expected_assembled_linted_code = textwrap.dedent(
```

Comment:
What do we need the dedent for here?

Reply:
It is true it is not necessary, but it was my swiss army knife for code strings, so I put it everywhere where a string was defined, in case we every moved something to an indented file. Will remove if you prefer, no strong optinion